misc/extension_api_validation/4.4-stable.expected

```diff
@@ -24,3 +24,12 @@ Validate extension JSON: Error: Field 'classes/OpenXRAPIExtension/methods/regist
 Validate extension JSON: Error: Field 'classes/OpenXRAPIExtension/methods/unregister_projection_views_extension/arguments/0': type changed value in new API, from "OpenXRExtensionWrapperExtension" to "OpenXRExtensionWrapper".
 
 Switched from `OpenXRExtensionWrapperExtension` to parent `OpenXRExtensionWrapper`. Compatibility methods registered.
+
+
+GH-86713
```

Comment:
These GDExtension changes are specific to this PR, not #86713, right?
```suggestion
GH-86716
```

Reply:
They are, it's a copy-paste blunder